docs/source/blueprints/BP000.rst

```diff
@@ -0,0 +1,307 @@
+   :Type: Epic Blueprint
+   :Status: WIP
+
+Blueprint statuses
+------------------
+
+ * Draft: All blueprints should be created in this state. This means the
+   blueprint is accepting comments, and probably there is a discussion
+   happening. Blueprints in draft mode can be part of our repository. 
+
+ * Approved: Blueprint was approved after discussions, and all suggestions are
+   already incorporated on the document. Nobody has started working on this
+   yet.
+
+ * WIP: This status is not about the blueprint itself, but about the proposal
+   that is the subject of the BP. This means that the blueprint was approved,
+   and someone is already working on implementing it. A BP status can change
+   from Draft to WIP if the work has started already. 
+
+ * Implemented: This means the BP is already implemented and delivered to the
+   Avocado’s master branch. 
+
+ * Rejected: Rejected status means the idea was not implemented because it
+   makes no sense or has some technical limitations.
+
+ * Deprecated: Deprecated means it was approved, implemented, and at some
+   point, makes no more sense to have it. For example, anything related to the
+   legacy runner. Usually, Deprecated means that it was replaced by something
+   else.
+
+As you can see, there is no status to accommodate any future change in a
+blueprint. Blueprints should not be “voided.” Any improvement on an old
+blueprint should be presented as a new blueprint, changing the status of the
+original to “deprecated”.
+
+Sections
+--------
+
+In order to facilitate the reading and understanding of the problem, all
+blueprints must have the following sections:
+
+ * TL;DR
+
+ * Motivation
+
+ * Specification
+
+ * Backwards Compatibility
+
+ * Security Implications
+
+ * How to Teach This
+
+ * Related Issues
+
+ * References
+
+Bellow a brief description of what you should write in each section:
+
+ * TL;DR: Should be a short description of your blueprint. Like an abstract. We
+   recommend writing this at the end of your first draft. This will give you a
+   better overview of it.
+
+ * Motivation: This should be the motivation of your proposed solution, not the
+   motivation of the blueprint itself. It describes the problem. Here, the
+   proposed solution should answer “why” this is needed.
+
+ * Specification: In this section, you should describe how you are going to
+   solve the problem. You can create subsections here to organize your ideas
+   better. Please keep in mind that it is useful to mention the details, with
+   code snippets, examples, and/or references. This will save you time, making
+   sure that everyone is in agreement with the proposed solution.
+
+ * Backwards Compatibility: How is your proposal going to affect older versions
+   of Avocado? Should we deprecate some modules, classes, or methods? Are we
+   going to keep backwards compatibility or not?
+
+ * Security Implications: What are your concerns about security with your
+   proposed solution?
```

Comment:
Maybe better since many people might have none: "Do you have any concerns about security with your proposed solution and what are they?"